custom-ops-ai-applications/pyproject.toml

```diff
@@ -28,4 +28,4 @@ fused_attention = { cmd = "python fused_attention.py", depends-on = ["package"]
 benchmarks = { cmd = "mojo benchmarks.mojo", depends-on = ["package"] }
 
 [tool.pixi.dependencies]
-max = "==25.2.0.dev2025031705"
+max = "==25.3.0.dev2025041105"
```

Comment:
Do we want to pin or make it wildcard from now?

Reply:
@ehsanmok without automated testing in place, we still need to pin